packages/dxf-deserializer/DxfReader.js

```diff
@@ -0,0 +1,253 @@
+/*
+## License
+
+Copyright (c) 2017 Z3 Development https://github.com/z3dev
+
+All code released under MIT license
+
+Thanks to @issacs for the sax js library, and inspiration for this reader
+
+*/
+
+// DXF reader to emit groups of interest to handlers
+
+;(function (dxf) { // wrapper for non-node envs
```

Comment:
this is strange: any reason why this is not structured as a 'normal' module ?

Reply:
Taken from the SAX parser. I’ll change to normal exports..

Reply:
Hmm... I stumped. I can make dxf into an object. Or I can export DxfReader. Ideally, dxf is just a name space. Suggestions?

Reply:
Hmm if it works like this , just leave it be for now, I'll take a closer look : I already did similar changes for AMF & SVG so we should align with that, but it is something for a next version I think :)